assets/css/style.css

```diff
@@ -1147,3 +1147,13 @@ pre code::-webkit-scrollbar-thumb {
     }
 
 }
+/* Changes to the footer and header links */
+
+.change-color-on-hover:hover{
+    text-decoration: underline !important;
+}
+.footer-social-icons:hover{
+    color: blue;
+    background-color: white;
```

Comment:
Instead of hard-coding the colors, use the user-configured colors that are embedded into the `head.html` stylesheet. There are some other examples of the `var(--colors)` being used in the CSS file. The color options are set in the config [here](https://github.com/unicef/inventory-hugo-theme/blob/main/exampleSite/config.yaml#L74-L81).